EIPS/eip-2410.md

```diff
@@ -0,0 +1,174 @@
+---
+eip: 2410
```

Comment:
```suggestion
eip: 7254
```
Assigning EIP number using PR number.  Please also update the file names.
EIP authors don't get to pick the EIP number.  I am not sure where 2410 came from?

Reply:
Please check it